src/components/Menu/Menu.js

```diff
@@ -1,24 +1,39 @@
 import React, { Component } from 'react';
```

Comment:
In the menu folder, logo.png is added as well, however this is never used since it's already added in the public/images folder. so better remove it here